browser/js/data/factory.board.js

```diff
@@ -26,7 +26,7 @@ app.factory('BoardFactory', function(GitHubFactory, CardFactory, $rootScope) {
 			var self = this;
 
 			//OP: ES6 => context remains same as outer scope, don't need self
-			return GitHubFactory.getRepo(currentBoard.githubID)
```

Comment:
@ssarda77 As you can see, I've changed githubID to be githubId throughout the project.  The capital D was tripping me up, so I just changed them.